src/sankoff_handler.cpp

```diff
@@ -62,11 +69,42 @@ EigenVectorXd SankoffHandler::TotalPPartial(NodeId node_id, size_t site_idx) {
          psv_handler_.GetPV(PSVType::PRight, node_id).col(site_idx);
 }
 
+void SankoffHandler::PopulateRootwardParsimonyPVForNode(const NodeId parent_id,
+                                                        const NodeId left_child_id,
+                                                        const NodeId right_child_id) {
+  for (size_t pattern_idx = 0; pattern_idx < site_pattern_.PatternCount();
+       pattern_idx++) {
+    // Which child partial is in right or left doesn't actually matter because they
+    // are summed when calculating q_partials. In DAG case, will want to ensure that
+    // left children partials are in p_partials_left
```

Comment:
This comment was just moved during a refactor.  This was either from Tanvi or Mary.  It can probably be removed.  I designate left and right children in the TPEngine.

Reply:
Let's clean it out.